src/Symfony/Installer/NewCommand.php

```diff
@@ -476,12 +494,46 @@ private function updateComposerJson()
             return $this;
         }
 
-        $ret = str_replace(
-            '"name": "symfony/framework-standard-edition",',
-            sprintf('"name": "%s",', $this->generateComposerProjectName()),
-            file_get_contents($filename)
+        $contents = json_decode(file_get_contents($filename), true);
+
+        $contents['name'] = $this->generateComposerProjectName();
+        $contents['license'] = 'proprietary';
+        unset($contents['description']);
+        unset($contents['extra']['branch-alias']);
+
+        file_put_contents($filename, json_encode($contents, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES)."\n");
+
+        return $this;
+    }
+
+    /**
+     * Creates the appropriate .gitignore file for a Symfony project.
+     *
+     * @return NewCommand
+     */
+    private function createGitIgnore()
+    {
+        $gitIgnoreEntries = array(
+            '/app/bootstrap.php.cache',
+            '/app/cache/*',
+            '!app/cache/.gitkeep',
+            '/app/config/parameters.yml',
+            '/app/logs/*',
+            '!app/logs/.gitkeep',
+            '/app/phpunit.xml',
+            '/bin/',
+            '/build/',
+            '/composer.phar',
```

Comment:
@xabbuh I'm afraid I don't understand you. Do you think we should remove this `createGitIgnore()` method from the installer?